stdlib/public/Python/NumpyConversion.swift

```diff
@@ -93,6 +93,13 @@ extension Double : NumpyScalarCompatible {
 
 extension Array : ConvertibleFromNumpyArray
   where Element : NumpyScalarCompatible {
+  /// Creates an `Array` with the same shape and scalars as the specified
+  /// `numpy.ndarray` instance.
+  ///
+  /// - Parameter numpyArray: The `numpy.ndarray` instance to convert.
+  /// - Precondition: The `numpy` Python package must be installed.
+  /// - Precondition: `numpyArray` must be 1-D and have a compatible scalar
+  ///   datatype.
```

Comment:
```suggestion
  ///   `dtype`.
```

Reply:
Done in https://github.com/apple/swift/pull/23069/commits/e7920e443eeb844ac662c91069d9c9025a3c2439.